### ISSUE TITLE
DropdownList - OnPush change detection strategy - popup placement 

### ISSUE THREAD

**Issue by moxival** [telerik/kendo-angular2#84]
When the DropdwonList (possibly other components are affected as well, but we haven't tested) is used within component that uses OnPush changedetection strategy some weird issue with popup placement occurs, please take a look at the plunker:

http://plnkr.co/edit/C9KNGre9pLKkDkucKQ8K?p=preview

**Comment by ggkrustev** [comment]
Hi @moxival,
Indeed the boundary detection wouldn't work when `onPush` strategy is used. The Popup component depends on a Angular lifecycle event to reposition when content is rendered, which will not be raised. 

After an extensive research we found out a solution solves the reposition issue. Unfortunately, there is a limitation in it. The animation direction would not be updated. Hence, if there the popup is flipped vertically the animation direction wouldn't be changed.

We will release a update of the Popup (probably tomorrow). Once it is out, you can just update the DropDowns package.

**Comment by moxival** [comment]
Hi @ggkrustev thank you for the detailed explanation and for the fast fix :)

**Comment by ggkrustev** [comment]
The latest release of Kendo Popup for Angular 2, version v0.13.1, should fix the 'onPush' issue.
I will close the issue for now. Please reopen it if the issue still persists.

**Comment by moxival** [comment]
Hey @ggkrustev i have a different problem with the popup now 😄, unfortunately i cannot give a plunker - how do i force Kendo Popup to be version 0.13.1 in plunker?

When the dropdown is in absolutely positioned container the Popup places itself in the left-top corner of the page (left: 0; top: 0;). The component holding the dropdownlist is still OnPush (default change detection strategy works as expected).

### Code example:
```typescript
import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector    : 'app-dashboard',
  template : `
<div style="position: absolute;top: 200px;left: 200px;z-index:100;">
      <kendo-dropdownlist [data]="['1','2','3']"></kendo-dropdownlist>
    </div>
`,
  changeDetection:ChangeDetectionStrategy.OnPush
})
export class DashboardComponent {
  constructor() {
  }
}
```
### Result
![image](https://cloud.githubusercontent.com/assets/323925/20382326/13120000-acb4-11e6-8a7c-6382b75c96e8.png)

**Comment by ggkrustev** [comment]
@moxival, you are absolutely right! We missed to `markForCheck` the updated popup position (silly me).

We pushed a fix and the latest Popup release, v0.13.2, fixes the issue (tested locally). Please give it a try and let if the issue still persists.
